homeassistant/components/openuv/config_flow.py

```diff
@@ -111,7 +111,7 @@ async def _async_verify(
             return self.async_show_form(
                 step_id=error_step_id,
                 data_schema=error_schema,
-                errors={CONF_API_KEY: "invalid_api_key"},
```

Comment:
This change seems odd; why report it less specifically?

Reply:
Even though it's not intended for use everywhere yet, I was looking at the `schema_config_entry_flow` helper and how it uses the `base` key no matter what: https://github.com/home-assistant/core/blob/377396ba166bb863927eacad436a13339802391f/homeassistant/helpers/schema_config_entry_flow.py#L229

It made me wonder if there was tactical value in using more specific keys. I'll follow your lead: if we should continue to be as specific as possible, I'll switch it back. 👍🏻

Reply:
Well, if the error is `base`, it will show up on just the top of the form.
If the error is the field name, it will show up on the field causing the error.

In this case, the error is an invalid API key, I think it is nice that it tells you that error next to the field that is wrong (and not just on top of the form).

That is at least how it was designed or meant to be 😓 

../Frenck